tests/model_explainability/lm_eval/conftest.py

```diff
@@ -307,3 +280,128 @@ def vllm_emulator_route(
         service=vllm_emulator_service.name,
     ) as route:
         yield route
+
+
+@pytest.fixture(scope="function")
+def lmeval_minio_pvc(
```

Comment:
please re-use `tests.model_serving.model_server.conftest.model_pvc` 
(in a separate pr also change the other pvc fixtures)